device/transport/amqp/src/amqp_ws.ts

```diff
@@ -4,21 +4,21 @@
 'use strict';
 
 import { Amqp } from './amqp.js';
-import { ClientConfig, Client, StableConnectionTransport } from 'azure-iot-device';
+import { Client } from 'azure-iot-device';
 
 /**
  * Constructs a transport object that can be used by the device {@link azure-iot-device.Client} to send and receive messages to and from an IoT Hub instance, using the AMQP protocol over secure websockets.
  * This class overloads the constructor of the base {@link azure-iot-device-amqp.Amqp} class from the AMQP transport, and inherits all methods from it.
  *
  * @augments module:azure-iot-device-amqp.Amqp
  */
-export class AmqpWs extends Amqp implements Client.Transport, StableConnectionTransport {
+export class AmqpWs extends Amqp implements Client.Transport {
```

Comment:
>implements Client.Transport [](start = 33, length = 27)

sillhy nit.  It inherits the Client.Transport implmementation from Amqp so this shouldn't be necessary